new version

Purple906 opened the pull request:
new version?

Purple906 committed:
removed old files added current

Purple906 committed:
adjusting some media queries

header is in good shape at the moment. grid is looking good, just need
actual content for each item. Footer is acting up still.

Purple906 committed:
updated changes to footer

need a little more adjusting but on to something.

Purple906 committed:
updated media queries and header

updated some media queries, currently everything looks mobile friendly.
head to foot. waiting on images for grid to possibly adjust layout.

Purple906 committed:
added assets and updates

added assets to the assets file and adjusted the header to remove the
hero image and fix the header.

Purple906 committed:
updating the header

adjusting title and others

Purple906 committed:
a little more work on the grid

seem to have finished the header, still need to adjust media queries.

adjusted grid and image sizes, still waiting on content from mom to
fill in.

Purple906 committed:
changing the images and layout

images all switched out and re-sized.

Purple906 committed:
added DDA calendar link

Purple906 committed:
fixed color and size of link for DDA

Purple906 committed:
ran prittify

Purple906 committed:
updating content

updating the content to reflect what mom wanted.

Purple906 committed:
removed the grid changes in media queries

small change to media queries. site is not responsive yet and with
adjustments to size of images it makes a little more sense this way.
only removed grid changes within media queries, all else stayed same.

mysterycommand committed:
chore: git merge origin master

mysterycommand committed:
chore: move Purple906's files over the old ones

mysterycommand committed:
Revert "chore: move Purple906's files over the old ones"

This reverts commit 93ed3a3676c48cb9e9dc67f9551b8fbbd3af30ab.

mysterycommand committed:
Revert "Revert "chore: move Purple906's files over the old ones""

This reverts commit cf0dd86f676818b257ef681b4000df6dba8466eb.

mysterycommand committed:
Merge pull request #5 from potpiedigital/mysterycommand/update-mvp

Revert "Revert "chore: move Purple906's files over the old ones""

mysterycommand committed:
chore: build social

Purple906 committed:
added new image to location to fit with other images

Purple906 committed:
trying to fix and make things responsive

changed width of hearing image, but cuts out the hours of operation.
probably need to move them anyway. feels weird in current location.

Purple906 committed:
moved store hours

still need to make responsive, but I think it fits better.

also need to update social media tags, because white ones have
disappeared.

Purple906 committed:
added location image to source assets

adjusting some content within grid to make responsive

Purple906 committed:
starting fresh

big clearance and redo on panara’s site. going to build it slow and
take my time so it doesn’t get out of control like the last 3 versions.

Purple906 committed:
did some good stuff

getting some sections moved along. currently avoiding location and
shopping section, not sure how to do the spacing and making everything
fit correctly and still be responsive. Blog and fair-trade sections
look good in full screen.

Purple906 committed:
updating some sizing

prepping for going responsive and the eventual media queries.

Purple906 committed:
playing with the nav

updating some of the spacing around the nav and title, trying to make
things line up with the content of the body a little more.

Purple906 committed:
updated images in assets, added images to shopping and location section

added all images and assets to all sections. Need to get copy from mom
to finalize sections and sayings. tried to fix shopping section and
“who made my cloths” overlay with z-index, but only seems to overlay
one of the images and won’t overlay the other two. Something is
missing/wrong.

mysterycommand committed:
Merge pull request #6 from potpiedigital/mysterycommand/build-social

Build and deploy nested assets.

mysterycommand committed:
chore: update the token, pipe push to /dev/null, and exit(1) if there's an error in deploy

mysterycommand committed:
Merge pull request #7 from potpiedigital/mysterycommand/fix-build

Fixes the Build

mysterycommand commented:
Hey dude, so I fixed the build and this should stay up to date at [http://panaraimports.com/staging/3/](http://panaraimports.com/staging/3/), but it doesn't look like the latest that you showed me … the last time we talked I think? Maybe you haven't sync'd?

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,125 @@
-<!--[if IE 8]>         <html lang="en-us" class="lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html lang="en-us"> <!--<![endif]-->
+<!DOCTYPE html>
+<html>
```
Please add `lang="en-US"`.

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,125 @@
+<head>
+    <title>Panara - Imports</title>
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
The head should probably look like this (you can remove the comments and the whitespace):
```html
<!-- these go first -->
<meta charset="utf-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">

<!-- these go together -->
<title>Panara - Imports</title>
<meta name="description" content="Unique fashions &amp; fabulous finds since 1992">

<!-- this (and any other head-specific script tags) can go last -->
<link rel="stylesheet" type="text/css" href="main.css">
```

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,125 @@
+        <h3 class="sub-title">UNIQUE FASHIONS &amp; FABULOUS FINDS SINCE 1992</h3>
+        <p class="email-phone">
+            <a href="mailto:panaramqt@gmail.com">panaramqt@gmail.com</a>
+            <span><a href="tel:906-226-4604">(906) 226-4604</a></span>
```
Might want to test this `tel` protocol. I think you had the area code in parenthesis in Poet's, and I think there's probably some rule about how the number should be formatted.

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,125 @@
+            <a href="mailto:panaramqt@gmail.com">panaramqt@gmail.com</a>
+            <span><a href="tel:906-226-4604">(906) 226-4604</a></span>
+        </p>
+        <p class="email-phone"> Mon - Fri 10-7 <span>Sat 12-7</span><span>Sun 12-4</span>
```
Why does this `p` have the class `.email-phone`?

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,125 @@
+        <div class="alpaca_sweater">
+            <img src="./assets/alpaca_sweater.png">
+            <p><strong>Open 7 days a week</strong></p>
+            <p><strong>362 Days a year!</strong></p>
```
Lowercase "days" … also, 362? Not 365? Closed, Xmas, NYs, & T-day?

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,125 @@
+            <div class="oils">
+                <img src="./assets/oils.png">
+                <p><strong>Esenial Oils and Incents</strong></p>
+                <p>available year round instore or online</p>
```
Splitting a sentence up over multiple `p` tags seems weird to me … oh, I see. It's probably more like the first `p` is actually a heading of some kind? Maybe `h5` or `h6` would be more appropriate?

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,125 @@
+        <div>
+            <div class="blog-post-1">
+                <img src="./assets/emma.png">
+                <h4>Persons of Interest</h4>
```
Oh, here's my `h4`!

mysterycommand reviewed (commented):
Looking pretty good over all. Couple of nits, but also I think I saw a more recent version of this the last time we talked? Looks like you need content from Mom, and it's not doing too hot in mobile at the moment.

Purple906 committed:
updates and making responsive

changed tag line, updated coloring of fonts, changed the background to
match in each section. added first media query.

Purple906 committed:
responsive fixes

updating the media query and some of the sizes to adjust for different
view ports. looks good, but some of the fonts still need work.

Purple906 committed:
updated the calendar and online store

added links to google calendar and online store.

Purple906 committed:
adjusting media query

Purple906 committed:
spacing in media query

mysterycommand reviewed (changes requested):
Hey, did you see these?

Purple906 committed:
added fonts

added fonts to CCS and the repo

Purple906 committed:
re-updating fonts

font-squirrel font generated fonts

Purple906 committed:
fixed fonts

Purple906 committed:
fixing the strong tags to be h2 tags

Purple906 committed:
small spacing adjustment

Purple906 committed:
changed some images

removed backgrounds

Purple906 committed:
fixed padding

Purple906 committed:
small updates to media query

working on finishing the responsiveness

Purple906 committed:
fixes to make it responsive

updating media queries to make more responsive adjusted some text as
well.

Purple906 committed:
more updates to media queries

adjusting font size  of p and h2,h3 tags and height of most of the
sections.

Purple906 committed:
final updates?

everything looks responsive on my end. did final adjustments to blog
section and finalized location and shopping. submitting for code review.

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
-                        <td>4:00 <abbr title="post meridiem">pm</abbr></td>
-                    </tr>
-                </table>
+    <title>Panara - Imports</title>
```
The hyphen here seems weird.

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
-                    </tr>
-                </table>
+    <title>Panara - Imports</title>
+    <meta name="description" content="A World of Different since 1991">
```
I think this should be `"A World of Different - Since 1991"`.

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+            <tr><img class="social-icon" src="./assets/social/pinterest-4-24.png"></tr>
+            <tr><img class="social-icon-1" src="./assets/social/yelp-2-24.png"></tr>
+        </table>
+        <p class="email-phone"> Mon - Fri 10-7 <span>Sat 12-7</span><span>Sun 12-4</span>
```
Why the space before `Mon` and around the hyphen in `Mon - Fri`? Also, why the `span`s around Sat/Sun hours?

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+    </section>
+    <section class="location">
+        <h1>Location</h1>
+        <h3>Great atmosphere very relaxed environment and the nicest changing rooms ever!</h3>
```
This is not a sentence.

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+    <section class="location">
+        <h1>Location</h1>
+        <h3>Great atmosphere very relaxed environment and the nicest changing rooms ever!</h3>
+        <p><a href="#">Directions</a><span>125 W Washington St Marquette MI 49855</span></p>
```
That link needs an `href`, and I think the address should be like `125 W. Washington St. Marquette, MI 49855` (note the punctuation).

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+        <h1>Location</h1>
+        <h3>Great atmosphere very relaxed environment and the nicest changing rooms ever!</h3>
+        <p><a href="#">Directions</a><span>125 W Washington St Marquette MI 49855</span></p>
+        <p><a href="https://calendar.google.com/calendar/embed?src=lssu.edu_vkq1adadc2soqa03nr70g0h11k%40group.calendar.google.com&ctz=America/New_York">Event Calendar</a></p>
```
We still doing the calendar thing? The `&` in this URL should be escaped (like `&amp;`) … kinda weird, but it should be.

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+        </div>
+        <div class="alpaca_sweater">
+            <img src="./assets/alpaca_sweater.png">
+            <h2>In Historic Downtown Marquette MI</h2>
```
`Marquette, MI`

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+    <img class="img-divider" src="./assets/landscape-1.png">
+    <section class="shopping">
+        <h1>Shopping</h1>
+        <h3>Can't get into the store? You can find most of your favorites like incense and body oils in our online store.</h3>
```
In a sentence like this, it's sometimes nice to use escaped apostrophe marks (like `&apos;`) instead of the single quote mark. Not a big deal obv. but it's a nice touch.

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+            </div>
+            <div class="oils">
+                <img src="./assets/oils.png">
+                <h2>Esenial Oils and Incents</h2>
```
`Essential oils and incense`

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+            <div class="oils">
+                <img src="./assets/oils.png">
+                <h2>Esenial Oils and Incents</h2>
+                <p>available year round instore or online</p>
```
`in store`

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+        <div>
+            <div class="blog-post-1">
+                <img src="./assets/emma.png">
+                <a href="http://panaraimports.tumblr.com/"><h4>Persons of Interest</h4></a>
```
The `a` tag should technically go inside the `h4` … might have to monkey with the CSS a bit to make the link "fill" the heading.

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+            <div class="blog-post-1">
+                <img src="./assets/emma.png">
+                <a href="http://panaraimports.tumblr.com/"><h4>Persons of Interest</h4></a>
+                <p>There are a lot of people in the world that inspire and represent what Panara is about, a world of different. I find celebrities and locals a like who follow their dreams push boundaries and always fight for what is best for their fellow man.</p>
```
`alike`

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+            <div class="blog-post-2">
+                <img src="./assets/great_lakes_jewlery.png">
+                <a href="http://panaraimports.tumblr.com/"><h4>Products of Interest</h4></a>
+                <p>I find new products, both nationally and locally that represent fair-trade sustainability and community development. Unfortunetly I don't have room for everything, find what I cant carry in the store through here.</p>
```
`I find new products, both nationally and locally, that represent fair-trade sustainability and community development. Unfortunately, I don&apos;t have room for everything. Find what I can&apos;t carry in the store here.`

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+        </div>
+        <div>
+            <img src="./assets/blog_hypnar.png">
+            <p>Folw my blog, if you feel so incline, and learn about what makes me passionate. Products I trust and reasons to always shop fair trade.</p>
```
`Follow my blog, if you feel so inclined. Learn about what makes me passionate, products I trust, and reasons to always shop fair-trade.`

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,129 @@
+            <img src="./assets/fairtrade_hypnar.png">
+            <p>Quality Products, Improving Lives, Protecting the Enviroment. Find information on Fair Trade and how it all works. Make a donation or offer support.</p>
+        </div>
+        <p><img src="./assets/copyright-32.png">. Pot Pie Digital. 2016</p>
```
`&copy; 2016, Pot Pie Digital`

mysterycommand reviewed (changes requested):
This is looking really good. Comments on spelling/grammar/etc, and a couple little HTML nit-picks. I still think fonts all look too small (especially the body/copy fonts) … I also think that almost all of it should be the same size: `3.8vw`. I got that number by dividing 100 by 26 … a lot of typography guides will tell you that line length should be "about one alphabet".

Also, I notice you doing a thing with padding and margin where you've got like left/bottom padding and then right margin. I think in most of these cases you could get away with a percent width and margin like:
```css
width: 80%;
margin: 0 auto 1em;
```

mysterycommand commented on source/main.css:
```diff
@@ -1,298 +1,561 @@
-    outline: thin dotted
+div > a {
+    color: #ffffff;
 }
```
Why?

mysterycommand commented on source/main.css:
```diff
@@ -1,298 +1,561 @@
+.blog-post-1 > a,
+.blog-post-2 > a {
+    color: #000000;
+    font-size: 1.5vw;
```
Too small.

mysterycommand commented on source/main.css:
```diff
@@ -1,298 +1,561 @@
-abbr[title] {
-    border-bottom: 1px dotted
+h2 {
+    font-size: 1.5vw;
```
Too small.

mysterycommand commented on source/main.css:
```diff
@@ -1,298 +1,561 @@
-    white-space: pre-wrap
+.location > h1,
+.shopping > h1,
+.blog > h1 {
```
Could this just apply to all `h1`s? Should there really be more than one `h1`?

mysterycommand commented on source/main.css:
```diff
@@ -1,298 +1,561 @@
-    quotes: "\201C" "\201D" "\2018" "\2019"
+.location > h3,
+.shopping > h3,
+.blog > h3 {
```
Could this just apply to all `h3`s?

mysterycommand commented on source/main.css:
```diff
@@ -1,298 +1,561 @@
-small {
-    font-size: 80%
+div > img,
+.fair-trade-hat {
```
Maybe just one class for drop shadows?

mysterycommand commented on source/main.css:
```diff
@@ -1,298 +1,561 @@
+.oils > img,
+.panara-shirt > img,
+.blog-post-1 > img,
+.blog-post-2 > img {
```
These are all so specific! Maybe just a single more generic class like `.img-container` or something?

mysterycommand commented on source/main.css:
```diff
@@ -1,298 +1,561 @@
+.panara-shirt > p,
+.oils > p,
+.blog-post-1 > p,
+.blog-post-2 > p {
```
… hmm, same as above?

mysterycommand reviewed (commented):
Okay, sorry, just reviewing the stylesheet now. I think you've got too many styles in here, lots more than you need really. I think you could get away with something more like:
```html
<body>
    <main>
        <header>
            <nav>
                <ul>
                    <li><a href=""></a></li>
                    <li><a href=""></a></li>
                    <li><a href=""></a></li>
                    <li><a href=""></a></li>
                </ul>
            </nav>
            <img src="" alt="">
            <h1></h1>
            <h4></h4>
            <div></div>
            <div></div>
            <div></div>
        </header>
        <section>
            <div class="content-container">
                <h2></h2>
                <p></p>
                <!-- optionally -->
                <img src="" alt="">
                <h3></h3>
                <p></p>
                <img src="" alt="">
            </div>
            <div class="image-container">
                <img src="" alt="">
                <img src="" alt="">
                <!-- optionally -->
                <img src="" alt="">
            </div>
        </section>
        <img src="" alt="" class="image-spacer">
        <!-- repeat section and .full-width-spacer a couple times -->
        <footer></footer>
    </main>
</body>
```

I think with that structure, you barely need class names at all. Maybe just for the `div`s? Maybe like this:
```css
html,
body {
    height: 100%;
    width: 100%;
}

body {
    background-color: #a2a2a2;
    margin: 0;
}

main {
    background-color: #e7e7e7;
    color: #444444;
    max-width: 960px;
    width: 100%;
}

header {
    align-items: center;
    display: flex;
    flex-direction: column;
    justify-content: center;
}

nav {
    width: 100%;
}

ul {
    align-items: center;
    display: flex;
    justify-content: space-around;
    list-style: none;
    margin: 0;
    padding: 0;
}

li {
    display: inline-block;
}

nav a {
    color: #ffffff;
    text-decoration: none;
}

section,
.image-spacer {
    width: 100%;
}

.content-container {
    width: 80%;
    margin: 0 auto 1em;
}

.image-container {
    align-items: center;
    display: flex;
    justify-content: space-around;
    width: 100%;
}

h1 {
    font-size: 10.5vw;
}

h2,
h3,
h4,
p {
    font-size: 4vw;
}
```

… anyway, I don't mean to tell you how to do your job, but that's where I'd start.

Purple906 committed:
cleaning up

Cleaning up all the CSS leftovers and mistakes from previous builds. I
really should have started fresh.

removed a lot of unneeded class names, changed section headers and ul’s

Purple906 committed:
adjusted all margin and padding

major adjustments to the margin and padding on most everything. Only on
full screen layout however, so need to redo all media queries.

Purple906 committed:
a lot of updates

completely re-wrote the media queries and adjusted some of the major
spacing, font size, margin and padding. Added a container div for the
floating images in location and shopping section to help things flow
better. Feels complete to me.

mysterycommand commented:
Hey dude, I hate to say it, but I think you need to work on the bigger media queries too now. It looks really good on a mobile device, but like what is going on here:
![screenshot 2017-01-24 21 23 10](https://cloud.githubusercontent.com/assets/810398/22275809/3342cafe-e27d-11e6-8489-ce06e70e6ba2.png)

Purple906 committed:
more updates to larger media queries

font size on some of the H tags, alpaca sweater div, major adjustments
to both alpaca and black jacket.

Purple906 committed:
adjustments to desktop size

adjusted and removed some old and un-needed code from desktop settings,
and then adjusted media queries as well.

Purple906 committed:
just forgot some links

like an idiot I forgot to include the a tag links for the images in the
footer. Also, fixed some lettering so everything is consistent.

Purple906 committed:
changed one image

changed the downtown mi image to be more fitting

Purple906 committed:
grammar

grammar fixes … it will be the death of me.

Purple906 committed:
last couple changes

This really really should be everything. I think its good to go public
… at least for now.

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,143 @@
+
+    <section id="blog">
+        <h1>Blog</h1>
+        <h3>My life, my store, my passions and random observations.</h3>
```
Just noticed that there's no link to the blog here?

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,143 @@
+        <div>
+            <div class="blog-post-1">
+                <img src="./assets/emma.png" alt="">
+                <h4><a href="http://panaraimports.tumblr.com/">Persons of Interest</a></h4>
```
Why would this be a link to the blog?

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,143 @@
+            </div>
+            <div class="blog-post-2">
+                <img src="./assets/great_lakes_jewlery.png" alt="">
+                <h4><a href="http://panaraimports.tumblr.com/">Products of Interest</a></h4>
```
Same here … what would make me think this is a link, and why would I expect these words to link to the blog. Will the blog have products in it? Could be pretty confusing for users.

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,143 @@
+    <img class="img-divider" src="./assets/landscape-3.png" alt="">
+
+    <section id="fair-trade">
+        <p>You can change the world with how you <a href="http://sustainableconnections.org/thinklocal/why">shop</a>. Learn more about <a href="http://fairtradeusa.org/">fair trade</a> and read stories about the people behind the <a href="http://panaraimports.tumblr.com/">products</a> we offer.</p>
```
Also here, why `products`?

mysterycommand commented on source/index.html:
```diff
@@ -1,77 +1,143 @@
+            <li><img src="./assets/social/facebook-3-24.png" alt=""></li>
+            <li><img src="./assets/social/instagram-24.png" alt=""></li>
+            <li><img src="./assets/social/pinterest-4-24.png" alt=""></li>
+            <li><img src="./assets/social/yelp-2-24.png" alt=""></li>
```
These should all be links right?

mysterycommand reviewed (changes requested):
I think it's looking better, the white space is still a little wonky to me, but it's probably good enough for a MVP. I noticed a couple issues with links and stuff and then I think we're good to go.

Purple906 committed:
updates

still need to add instagram and pinterest, but links should all be
updated

Purple906 committed:
removed dead links

Removed all link to non-real things. no blog, no Instagram, no
pinterest, etc.

Purple906 committed:
put nav bar blog in

put nav bar blog back in since it just goes to the blog section or the
website.

Purple906 committed:
bye bye blog

but the content stays.

Purple906 committed:
redirected blog image in footer

mysterycommand commented:
Hey, sorry about this, I think probably just remove/comment-out the "Blog" links in the nav/header and the footer. That section doesn't make sense to go to, but kind of does stand on it's own content-wise … up to you if you want to remove the `id="blog"` from the section.

Purple906 committed:
removed blog from nav and footer

commented out blog tag in both nav and footer. leaving in code to
re-establish if blog is started and active.

Purple906 commented:
are we good? would love to let mom know the site is up ... so she can immediately start to give me feedback of things she wants changed.

mysterycommand commented:
Okay, good to go bro. Click that "Merge pull request" button and then go ahead and delete the branch. It should update panaraimports.com in a couple minutes. 👍
